<launch>
    <node name="cam_transformer" pkg="simple_camera_transformer" type="cam_transformer.py" respawn="true" respawn_delay="10" output="screen">
        <param name="pub_topic"   type="string" value="/usb_cam/image_transformed" />
        <param name="sub_topic" type="string" value="/usb_cam/image_raw" />
    </node>
</launch>
